vendor/bitbucket.org/liamstask/goose

```diff
@@ -0,0 +1 @@
+Subproject commit 8488cc47d90c8a502b1c41a462a6d9cc8ee0a895
```

Comment:
@Kirikami DO NOT commit `./vendor` folder, this should be in `.gitignore` You can install all dependencies using `glide install`